tests/.ignored-deprecations-legacy-events

```diff
@@ -25,3 +25,4 @@
 %ApiPlatform\\Api\\FilterInterface is deprecated in favor of ApiPlatform\\Metadata\\FilterInterface%
 
 %The "Symfony\\Bundle\\MakerBundle\\Maker\\MakeAuthenticator" class is deprecated, use any of the Security\\Make\* commands instead%
+%Since symfony/validator 7.1: Not passing a value for the "requireTld" option to the Url constraint is deprecated. Its default value will change to "true".%
```

Comment:
nit: we could pass the value explicitly instead?

Reply:
not really:
1. it's only in our tests
2. we need to find a way to know if the option exists
3. the default value is fine for us (as not really used by us)

therefore it won't break when updating